components/home/menu/calendar/Screens/DashboardScreen/index.js

```diff
@@ -0,0 +1,108 @@
+import React, { Component } from 'react';
+import { View, Text, TouchableOpacity, StyleSheet, Alert } from 'react-native';
+import { Agenda } from 'react-native-calendars';
+import styles from './styles';
+
+export default class DashboardScreen extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      items: {},
+      synchronizedEvents: this.structureSynchronizedEvents(
+        props.navigation.state.params.jsonFile.items
+      )
+    };
+  }
+
+  structureSynchronizedEvents(events) {
+    let tempArray = [];
+    events.forEach(event => {
+      tempArray.push(
+        {
+          date: event.start.dateTime != null ? event.start.dateTime.substring(0,event.start.dateTime.indexOf('T')):event.start.date,
+          title: event.summary!=null ? event.summary:'No Title For this Event',
+          startTime: event.start.dateTime != null ? event.start.dateTime : event.start.date,
+          endTime: event.end.dateTime != null ? event.end.dateTime : event.end.date,
+          description: event.description != null ?event.description : '',
+          address: ''
+        }
+      )
+    });
+    this.setState({
+      synchronizedEvents: this.tempArray
+    });
+    console.log('temp:\n', tempArray, '--->\n\n\-n>>>>');
```

Comment:
Remove all console.log's. You have a bunch more of these in other files.

Reply:
All console.log statements are removed now.